authors/apps/authentication/serializers.py

```diff
@@ -176,3 +179,22 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ResetEmailSerializer(serializers.ModelSerializer):
+    """
+    Serializer for requesting a password reset e-mail.
+    """
+    email = serializers.EmailField()
+
+    @classmethod
+    def validate_email(self, value):
+        try:
+            valid_email = User.objects.get(email=value)
```

Comment:
@ManuelDominic , here you could make use of the model method `get_object_or_404`